templates/AureliaSpa/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "AureliaSpa",
+  "version": "0.0.0",
+  "aurelia": {
+    "build": {
+      "includeDependencies": "aurelia-*"
+    }
+  },
+  "dependencies": {
+    "@types/bluebird": "^3.0.35",
+    "@types/node": "^6.0.45",
+    "aurelia-bootstrapper-webpack": "^1.0.0",
+    "aurelia-event-aggregator": "^1.0.0",
+    "aurelia-fetch-client": "^1.0.0",
+    "aurelia-framework": "^1.0.0",
+    "aurelia-history-browser": "^1.0.0",
+    "aurelia-loader-webpack": "^1.0.0",
+    "aurelia-logging-console": "^1.0.0",
+    "aurelia-pal-browser": "^1.0.0",
+    "aurelia-polyfills": "^1.0.0",
+    "aurelia-route-recognizer": "^1.0.0",
+    "aurelia-router": "^1.0.2",
+    "aurelia-templating-binding": "^1.0.0",
+    "aurelia-templating-resources": "^1.0.0",
+    "aurelia-templating-router": "^1.0.0",
+    "bluebird": "^3.4.1",
+    "bootstrap": "^3.3.7",
+    "isomorphic-fetch": "^2.2.1",
+    "jquery": "^2.2.1",
+    "preboot": "^4.5.2"
+  },
+  "devDependencies": {
```

Comment:
I'm not sure it's valid to regard these as `devDependencies`. When you're publishing to production, isn't the publish script set up so that it would only restore `dependencies`, and then the webpack compilation would fail?